README.txt
site simples html


CHANGELOG.txt
Date

2008/05/17
	* Changed password.php and changemail.php to POST method. [BeoWulf]
	* Fixed the mail sending function, now the recover password function should work, also
	now that we're using the phpmailer class you should be able to use gmail smtp server too [Dekamaster]
	* Disabled error report by default. [Dekamaster]
	* Added a definition file for the e-mail template, now you can customize the template of
	the email that will be sent to your players in /language/mail/ [Dekamaster]
	* Added Castle names, kisuka report on eAthena forums. [BeoWulf]
	* Added extra protection to login.php after some security leak rumors (not yet confirmed though). [BeoWulf]
	* Testing a new Captcha, its uglier than the original, but more safe. [Dekamaster]
	* Extended some of login.php protection to account.php and recover.php just in case. [BeoWulf]

2008/05/14
	* Fixed a bug with reset position leting a jailed person to get out, reported by panchox
	at ea boards (and thanks for the temporary fix, brianlew, we really appreciated it.) [Dekamaster]

------------------
We are retaking the project today (2008/05/14), we'll be slowly adding new stuffs to the control
panel and fixing old bugs, if you have a bug to report you are welcome to share it with us at
cerescp@gmail.com (yes we are reading it again)
------------------

2007/01/01
	* Fixed Firefox incompatibility, stupid browser. [BeoWulf]

2006/12/03
	* Reviewed img.php and middle.php files with help of gunawan wibisono. [BeoWulf]

2006/10/23
	* Added a variable in recover.php. [BeoWulf]

2006/10/18
	* Fixed an exploit in language.php. [BeoWulf]

2006/10/15
	* Added a one day ipban to users blocked, it will keep then out to think about
	what they did and can't creat accounts with ofensive nicks against GMs. [BeoWulf]
	* Small fix in server_status.php, and a change in account.php so now its
	post not get (internal stuff). [BeoWulf]

2006/10/03
	* Fixed errors in emblems.php, adminquery.php, ladder.php and functions.php. [BeoWulf]

2006/10/03
	* Changed all session maybe now there is no conflict with 2 cps in same machine. [BeoWulf]

2006/09/22
	* Added some fuzzy to code image. [BeoWulf]

2006/09/18
	* Created first trial of an install script. [BeoWulf]
	* Fixed install script checks and w3c instructions. [BeoWulf]
	* Changed INSTALL.TXT instructions to fit the new install script. [BeoWulf]

2006/09/18
	* Added jail check to position.php so jailed characters may not reset
		their position. [Entwined]
	* Added Post functions do ajax engine. [BeoWulf]
	* Changed login.php from get to post. [BeoWulf]
	* Fixed whoisonline.php adjust to show position to minimum admin level set 
		in config.php. [BeoWulf]
	* Added check in functions.php to destroy session if you use more than one
		CP in same machine when changing beteween them use different names 
		for the CPs or it can be exploitable, still working on a different 
		solution. [BeoWulf]
	* Changed admincharinfo.php to a beta test in changing stored char_id in 
		equip slots into something we can read. [BeoWulf]
	* Added adminchars.php still working on it, to manage chars. [BeoWulf]
	* Added money.php testing a 2 minutes ban after transfer. [BeoWulf]
	* Added marriage.php testing a 2 minutes ban after divorce. [BeoWulf]
	* Added guild.php if castle name is not defined it will skip and not show. [BeoWulf]
	* Added dutch and indonesian to languages. [BeoWulf]
	* Fixed some stuff in other language files, but still need translations. [BeoWulf]
	* Fixed ladder.php and about.php to show the new classes. [BeoWulf]

2006/09/14
	* Added map and gm level check for position reset, so you can't reset your position if your in jail [Entwined]
	* Added install/sql/rag_ragsrvinfo.sql and install/npc/agit_status.txt. You will have to install both of 
		these correctly for agit status	check to work. [Entwined]
	* Added ceres.php for control panel information [Entwined]
	* Added max account check for account registration [Entwined]

2006/09/13
	* Added IP address field insertion on registrations (last_ip) [Entwined]
	* Added an email verification field for account registration. [Entwined]
	* Added some config settings for future use. [Entwined]
	* Created the initial repository of CeresCP (rev4) as the working base. [Entwined]


info.php
<?php 
extension_loaded('mysqli')

	or die ("Mysqli extension not loaded. Please verify your PHP configuration.");



is_file("./config.php")

	or die("<a href=\"./install/install.php\">Run Installation Script</a>");



session_start();

include_once 'config.php'; // loads config variables

include_once 'query.php'; // imports queries

include_once 'functions.php';



$_SESSION[$CONFIG_name.'castles'] = readcastles();

$_SESSION[$CONFIG_name.'jobs'] = readjobs();



?>
<!--
	Dopetrope by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>RagnaDream</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<script type="text/javascript" language="javascript" src="ceres.js"></script>
	</head>
	<body class="left-sidebar is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<section id="header">
<img src="images/menu.png" width="60%"  style = "padding-left:1px;"></img>
					<!-- Logo -->
						

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="indexpg.php">Home</a></li>
								<li ><a href="cadastro.php">Cadastro</a></li>
								<li class="current"><a href="info.php">Informação</a></li>
								
							</ul>
						</nav>

				</section>

			<!-- Main -->
				<section id="main">
					<div class="container">
						<div class="row">
							<div class="col-4 col-12-medium">

								<!-- Sidebar -->
									<section class="box">
									<div id="login_div" style="padding-left: 15; padding-right: 5; padding-top: 10">

								</div>
									<script type="text/javascript">


			LINK_ajax('login.php', 'login_div');

			login_hide(1);

		</script>
								<!----- inicio do login --->
								
								<div id="main_menu" style="color:#FFFFFF; font-weight:bold"></div>

					<div id="load_div" style="visibility:hidden; background-color:#000000; color:#FFFFFF"><center><img src="images/loading.gif" alt="Loading..."></center></div>

					<div id="menu_load" style=" visibility:hidden;"></div>

			<tr valign=top>

				<td height="100%">

					<table border="0" >

						<tr >

							<td valign="top" bgcolor="#FFFFFF" colspan="2"  >

						

							</td>

							<td class="back_green" valign="top" >

								<div id="login_div" style="margin-right: 20%;">

								</div>

								<div id="new_div" >

								</div>

								<div id="status_div" >

								</div>

								<div id="selectlang_div" >

								</div>

							</td>

						</tr>

					</table>


		<script type="text/javascript">


			LINK_ajax('login.php', 'login_div');

			login_hide(1);

			server_status();
			hidecadastro();

		</script>
								
								<!--- fim do login--->
									</section>
									

							</div>
							<div class="col-8 col-12-medium imp-medium">

								<!-- Content -->
									<article class="box post">
										<!----> Content<a href="#" class="image featured"><img src="images/info.jpg"  /></a> 
										<header>
											<h2> Informações <img src="/images/anubis.gif" alt="" sizes="15" width="70" srcset=""></h2>
										</header>
										<p><table id="tablePreview" class="table">
											<!--Table head-->
											<thead>
												<tr>
												<th></th>
												<th></th>
											
												</tr>
											</thead>
											<!--Table head-->
											<!--Table body-->
											<tbody>
												<tr>
												<th scope="row">Rates</th>
												<td>30x/20x/7x</td>
												
												</tr>
												<tr>
												<th scope="row">Rates Quest</th>
												<td>9x</td>
												</tr>
												<tr>
												<th scope="row">EXP extra por membro no grupo</th>
												<td>5%</td>
												</tr>
												<tr>
												<th scope="row">rate card</th>
												<td>0.06%</td>
												
												</tr>
												<tr>
												<th scope="row">rate MVP card</th>
												<td>0.01%</td>
												</tr>
											</tbody>
											<!--Table body-->
											</table>
											<table id="tablePreview" class="table">
											<!--Table head-->
											<thead>
												<tr> <h3>Comandos/ Npc<h3>
												<th></th>
												<th></th>
											
												</tr>
											</thead>
											<!--Table head-->
											<!--Table body-->
											<tbody>
												<tr>
												<th scope="row">@go</th>
												<td>teleporta para cidades de uma lista.</td>
												
												</tr>
												<tr>
													<th scope="row">@rates</th>
													<td>Consulta atual rate do serve.</td>
												</tr>
												<tr>
													<th scope="row">Lista de NPC</th>
													<td></td>
												</tr>
												<tr>
													<th scope="row">Corrida poring</th>
													<td>Devil squard</td>
												</tr>
												<tr>
													<th scope="row">MVP point</th>
													<td>Presenca e muito mais...</td>
												</tr>
												
												
											</tbody>
											<!--Table body-->
											</table>

											<div id="main_menu" style="color:#FFFFFF; font-weight:bold"></div>
											<div id="main_div" style="padding-left: 15; padding-right: 5; padding-top: 10">

								</div>

										</p>
										
										
										
									</article>

							</div>
						</div>
					</div>
				</section>

			<!-- Footer -->
				<section id="footer">
					<div class="container">
						<div class="row">
							<div class="col-8 col-12-medium">
								<section>
									<header>
										<h2>Blandit nisl adipiscing</h2>
									</header>
									<ul class="dates">
										<li>
											<span class="date">Jan <strong>27</strong></span>
											<h3><a href="#">Lorem dolor sit amet veroeros</a></h3>
											<p>Ipsum dolor sit amet veroeros consequat blandit ipsum phasellus lorem consequat etiam.</p>
										</li>
										<li>
											<span class="date">Jan <strong>23</strong></span>
											<h3><a href="#">Ipsum sed blandit nisl consequat</a></h3>
											<p>Blandit phasellus lorem ipsum dolor tempor sapien tortor hendrerit adipiscing feugiat lorem.</p>
										</li>
										<li>
											<span class="date">Jan <strong>15</strong></span>
											<h3><a href="#">Magna tempus lorem feugiat</a></h3>
											<p>Dolore consequat sed phasellus lorem sed etiam nullam dolor etiam sed amet sit consequat.</p>
										</li>
										<li>
											<span class="date">Jan <strong>12</strong></span>
											<h3><a href="#">Dolore tempus ipsum feugiat nulla</a></h3>
											<p>Feugiat lorem dolor sed nullam tempus lorem ipsum dolor sit amet nullam consequat.</p>
										</li>
										<li>
											<span class="date">Jan <strong>10</strong></span>
											<h3><a href="#">Blandit tempus aliquam?</a></h3>
											<p>Feugiat sed tempus blandit tempus adipiscing nisl lorem ipsum dolor sit amet dolore.</p>
										</li>
									</ul>
								</section>
							</div>
							<div class="col-4 col-12-medium">
								<section>
									<header>
										<h2>What's this all about?</h2>
									</header>
									<a href="#" class="image featured"><img src="images/pic10.jpg" alt="" /></a>
									<p>
										This is <strong>Dopetrope</strong> a free, fully responsive HTML5 site template by
										<a href="http://twitter.com/ajlkn">AJ</a> for <a href="http://html5up.net/">HTML5 UP</a> It's released for free under
										the <a href="http://html5up.net/license/">Creative Commons Attribution</a> license so feel free to use it for any personal or commercial project &ndash; just don't forget to credit us!
									</p>
									<footer>
										<ul class="actions">
											<li><a href="#" class="button">Find out more</a></li>
										</ul>
									</footer>
								</section>
							</div>
							<div class="col-4 col-6-medium col-12-small">
								<section>
									<header>
										<h2>Tempus consequat</h2>
									</header>
									<ul class="divided">
										<li><a href="#">Lorem ipsum dolor sit amet sit veroeros</a></li>
										<li><a href="#">Sed et blandit consequat sed tlorem blandit</a></li>
										<li><a href="#">Adipiscing feugiat phasellus sed tempus</a></li>
										<li><a href="#">Hendrerit tortor vitae mattis tempor sapien</a></li>
										<li><a href="#">Sem feugiat sapien id suscipit magna felis nec</a></li>
										<li><a href="#">Elit class aptent taciti sociosqu ad litora</a></li>
									</ul>
								</section>
							</div>
							<div class="col-4 col-6-medium col-12-small">
								<section>
									<header>
										<h2>Ipsum et phasellus</h2>
									</header>
									<ul class="divided">
										<li><a href="#">Lorem ipsum dolor sit amet sit veroeros</a></li>
										<li><a href="#">Sed et blandit consequat sed tlorem blandit</a></li>
										<li><a href="#">Adipiscing feugiat phasellus sed tempus</a></li>
										<li><a href="#">Hendrerit tortor vitae mattis tempor sapien</a></li>
										<li><a href="#">Sem feugiat sapien id suscipit magna felis nec</a></li>
										<li><a href="#">Elit class aptent taciti sociosqu ad litora</a></li>
									</ul>
								</section>
							</div>
							<div class="col-4 col-12-medium">
								<section>
									<header>
										<h2>Vitae tempor lorem</h2>
									</header>
									<ul class="social">
										<li><a class="icon brands fa-facebook-f" href="#"><span class="label">Facebook</span></a></li>
										<li><a class="icon brands fa-twitter" href="#"><span class="label">Twitter</span></a></li>
										<li><a class="icon brands fa-dribbble" href="#"><span class="label">Dribbble</span></a></li>
										<li><a class="icon brands fa-tumblr" href="#"><span class="label">Tumblr</span></a></li>
										<li><a class="icon brands fa-linkedin-in" href="#"><span class="label">LinkedIn</span></a></li>
									</ul>
									<ul class="contact">
										<li>
											<h3>Address</h3>
											<p>
												Untitled Incorporated<br />
												1234 Somewhere Road Suite<br />
												Nashville, TN 00000-0000
											</p>
										</li>
										<li>
											<h3>Mail</h3>
											<p><a href="#">someone@untitled.tld</a></p>
										</li>
										<li>
											<h3>Phone</h3>
											<p>(800) 000-0000</p>
										</li>
									</ul>
								</section>
							</div>
							<div class="col-12">

								<!-- Copyright -->
								<div id="copyright">
										<ul class="links">
											<li>&copy; RagnaDream.</li><li>Design: <a href="#">Augusto Perez</a></li>
										</ul>
									</div>

							</div>
						</div>
					</div>
				</section>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>